metricflow/test/fixtures/model_fixtures.py

```diff
@@ -68,6 +68,23 @@ def query_parser_from_yaml(
     )
 
 
+@pytest.fixture(scope="session")
+def query_parser(  # noqa: D
+    simple_semantic_model: SemanticModel,
+    consistent_id_object_repository: ConsistentIdObjectRepository,
+    time_spine_source: TimeSpineSource,
+) -> MetricFlowQueryParser:
+    return MetricFlowQueryParser(
+        model=simple_semantic_model,
+        source_nodes=consistent_id_object_repository.simple_model_source_nodes,
+        node_output_resolver=DataflowPlanNodeOutputDataSetResolver(
+            column_association_resolver=DefaultColumnAssociationResolver(simple_semantic_model),
+            semantic_model=simple_semantic_model,
+            time_spine_source=time_spine_source,
+        ),
+    )
+
```

Comment:
The reason this was removed was because it's doing a bunch of stuff against stored models and that makes these tests very rigid. The whole file was actually cut over to focus pretty much exclusively on parsing and processing a simple local model directly illustrating the behavior we need to test.

Can we update the new test case added in #370 to bring it in line with the rest of the tests in the test_query_parser module?

Reply:
gotcha, i updated it to not use the stored models